### PR TITLE
update release-1.6.14 release note

### PR DESCRIPTION
**Pull request by yangminzhu**
[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[x] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by mandarjog — APPROVED** [review]
lgtm

File: content/en/news/releases/1.6.x/announcing-1.6.14/index.md [hunk 1]
```diff
@@ -19,4 +19,5 @@ This release contains bug fixes to improve robustness. This release note describ
   ([Issue #28916](https://github.com/istio/istio/issues/28916))
 - **Fixed** an issue that caused very high memory usage with a large number of `ServiceEntries`.
   ([Issue #25531](https://github.com/istio/istio/issues/25531))
-
+- **Fixed** an issue in the Stackdriver access log of the `useragent` header.
```
Comment (ericvn) [inline content/en/news/releases/1.6.x/announcing-1.6.14/index.md:22]: I'm not sure this is quite correct. Does Fixed an issue with the `user agent` header in the Stackdriver access log sound correct?

Comment (yangminzhu) [reply content/en/news/releases/1.6.x/announcing-1.6.14/index.md]: Updated.

**Comment by yangminzhu** [comment]
@ericvn Updated a little bit, could you take a look again, thanks.

File: content/en/news/releases/1.6.x/announcing-1.6.14/index.md [hunk 2]
```diff
@@ -19,4 +19,5 @@ This release contains bug fixes to improve robustness. This release note describ
   ([Issue #28916](https://github.com/istio/istio/issues/28916))
 - **Fixed** an issue that caused very high memory usage with a large number of `ServiceEntries`.
   ([Issue #25531](https://github.com/istio/istio/issues/25531))
-
+- **Fixed** an issue that caused the `user agent` header missing in the Stackdriver access log.
```
Comment (ericvn) [inline content/en/news/releases/1.6.x/announcing-1.6.14/index.md:22]: Maybe: Fixed an issue that caused the `user agent` header to be missing in the Stackdriver access log.

Comment (yangminzhu) [reply content/en/news/releases/1.6.x/announcing-1.6.14/index.md]: @ericvn updated, thanks.

**Review by ericvn — APPROVED** [review]

**Pull request merged**